pkg/network/tracer/tracer_windows_test.go

```diff
@@ -27,5 +27,5 @@ func httpsSupported(t *testing.T) bool {
 }
 
 func classificationSupported(config *config.Config) bool {
-	return false
+	return true
```

Comment:
```suggestion
	return config.ProtocolClassificationEnabled
```

Reply:
That's not how those flags are (supposed) to work.
It is now _supported_, which indicates for the test whether the tests should be run or not at all.

Reply:
I copied the logic from httpsSupported...
But im ok eith the current impl